content/en/containers/kubernetes/installation.md

```diff
@@ -158,7 +158,7 @@ Using the Datadog Operator requires the following prerequisites:
 - `<TARGET_SYSTEM>`: The name of your OS. For example, `linux` or `windows`.
 
 
-**Note**: If you are using Helm < v3, run the following:
+**Note**: If you are using Helm v2.x, run the following:
```

Comment:
I feel like we usually use `2.x` with no v for a version number, but I leave it up to you, since it was already like that and I'm not entirely sure. :)

Reply:
You are right, good catch! I like that better too.